app/components/institution-page/campus-program-application-modal.ts

```diff
@@ -27,6 +31,19 @@ export default class CampusProgramApplicationModal extends Component<Signature>
         .sortBy('createdAt').lastObject || null
     );
   }
+
+  @action
+  handleEnterEmailStepSubmitted() {
+    this.forceShowEnterEmail = false;
+    this.emailAddressToPrefill = undefined;
+  }
+
+  @action
+  handleGoBackToEnterEmail() {
+    this.emailAddressToPrefill = this.latestSavedGrantApplication!.originalEmailAddress;
```

Comment:
Let's use `normalizedEmailAddress` instead of `originalEmailAddress`. The verify email step renders `normalizedEmailAddress` so that's what people would expect to see here.